ajax/libs/kbw.svg/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "kbw.svg",
+  "filename": "jquery.svg.min.js",
+  "description": "This plugin allows you to control/interact with SVG elements within your HTML documents. * Native SVG support in major browsers (but only IE9+). * Create a new SVG element or wrap an existing inline SVG element. * Animate SVG attributes with the svganim module. * Apply jQuery class/attribute/event functionality to SVG elements with the svgdom module. * Draw graphs with the svggraph module. * Plot functions with the svgplot module.",
+  "keywords": [
+    "svg",
+    "canvas",
+    "graphics",
+    "graph",
+    "chart",
+    "plot",
+    "ui"
+  ],
+  "version": "1.5.0",
+  "author": {
+    "name": "Keith Wood",
+    "url": "http://keith-wood.name/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kbwood/svg"
+  },
+  "license": "MIT",
+  "homepage": "http://keith-wood.name/svg.html",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kbwood/svg.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.svg.*"
```

Comment:
I think we can use `jquery.svg*`, as the homepage said in page `Extension Plugins`.
What do you think?

Thank you.

Reply:
@sufuf3, thanks your suggestion. I think `jquery.svg*` is inappropriate, because it will fetch the files we do't need.  
For example, `jquery.svgdom.js`.

Reply:
@nolsherry The homepage said we need them.
http://keith-wood.name/svg.html --> `Extension Plugins`